packages/bpk-component-calendar/src/BpkCalendarDate.scss

```diff
@@ -60,6 +60,7 @@
   }
 
   &--outside {
+    background-color: transparent;
```

Comment:
Added as when the custom styles were applied even if  the date was out of range it would show them as colored